tools/cloudflow-operator/src/main/scala/cloudflow/operator/action/PrepareNamespaceActions.scala

```diff
@@ -16,21 +16,25 @@
 
 package cloudflow.operator.action
 
-import scala.collection.immutable._
-import skuber._
+import akka.datap.crd.App
+import akka.kube.actions.Action
 import cloudflow.operator.action.runner.Runner
+import io.fabric8.kubernetes.api.model.OwnerReference
+
+import scala.collection.immutable._
 
 /**
  * Creates a sequence of resource actions for preparing the namespace where the application is
  * installed
  */
 object PrepareNamespaceActions {
-  def apply(app: CloudflowApplication.CR,
-            runners: Map[String, Runner[_]],
-            labels: CloudflowLabels,
-            ownerReferences: List[OwnerReference]): Seq[Action] =
+  def apply(
+      app: App.Cr,
+      runners: Map[String, Runner[_]],
+      labels: CloudflowLabels,
+      ownerReferences: List[OwnerReference]): Seq[Action] =
     app.spec.streamlets
-      .map(streamlet => streamlet.descriptor.runtime.name)
+      .map(streamlet => streamlet.descriptor.runtime)
```

Comment:
Why was this change needed?

Reply:
it's related to the usage of the Fabric8 CR